oat-sa/tao-test-runner-qti-fe
# Fix/actp 410/tools keyboard navigation

**Related to**
https://oat-sa.atlassian.net/browse/ACTP-410

**Description**
Sometimes it was't possible to navigate through tools menu via keyboard. This happened in case when first item was hidden.

**How to test**
1. Create a test with tools.
2. Start a test.
3. Try to navigate using up and down arrows.
4. Check different combination of tools.
